classes/publication/PKPPublication.php

```diff
@@ -149,7 +150,7 @@ public function getTitles()
      *
      * @return string
      */
-    public function getAuthorString($userGroups, $includeInBrowseOnly = false)
+    public function getAuthorString(LazyCollection $userGroups, $includeInBrowseOnly = false)
```

Comment:
- I changed the `@param` to `\Traversable<UserGroup>` - I am trying to use something like "`Generics`" in documentation where ever I can.
- I changed the function's signature to `\Traversable`

I understand that the `\Traversable` is more descriptive for the actual use of the `$userGroups` inside the function (only in `foreach` clause), and can lead to less issues regarding possible fatal errors that may appear because we pass an `array` for example, instead of a `LazyCollection`. I used `LazyCollection`, though, to force the use of a `Collection` object (for the reasons you described above), and also to be able to use the functions and possibilities of a `[Collection] (https://laravel.com/docs/9.x/collections)` if necessary.